### PR TITLE
Package Kozani Artifacts in MSIX and Nuget packages

### PR DESCRIPTION
**Pull request by sachintaMSFT**
Package Kozani artifacts in MSIX and Nuget packages. Along with aggregator pipeline change in PR, this change will ensure Kozani binaries are packed into Framework and Singleton packages, appropriately.

### CODE REVIEW

File: build/CopyFilesToStagingDir.ps1 [hunk 1]
```diff
@@ -55,6 +55,27 @@ PublishFile $FullBuildOutput\RestartAgent\RestartAgent.exe $FullPublishDir\Micro
...
+PublishFile $FullBuildOutput\KozaniAppGraph\StrippedWinMD\Microsoft.Kozani.AppGraph.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniPackage\StrippedWinMD\Microsoft.Kozani.Package.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\StrippedWinMD\Microsoft.Kozani.SendToLocal.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\StrippedWinMD\Microsoft.Kozani.SendToRemote.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\StrippedWinMD\Microsoft.Kozani.Settings.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniManagerRuntime\StrippedWinMD\Microsoft.Kozani.ManagerRuntime.winmd $FullPublishDir\Microsoft.WindowsAppRuntime\
+#
+### Kozani DLLs/EXEs in MSIX Framework package
+PublishFile $FullBuildOutput\KozaniAppGraph\KozaniAppGraph.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniAppGraph\KozaniAppGraph.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
```
Comment (cwruss) [inline build/CopyFilesToStagingDir.ps1:70]: Why do the PDBs get published into the fwk package? Is that required for debugging?

Comment (sachintaMSFT) [reply build/CopyFilesToStagingDir.ps1:70]: This repo only feeds a transport package to aggregator pipeline. Aggregator repo's pipeline includes only .dll and specific .exe files in producing the framework MSIX packages - this change is internal PR I am sending out shortly.

**Review by cwruss — APPROVED** [review]

**Review by DrusTheAxe — APPROVED** [review]

File: build/CopyFilesToStagingDir.ps1 [hunk 2]
```diff
@@ -55,6 +55,27 @@ PublishFile $FullBuildOutput\RestartAgent\RestartAgent.exe $FullPublishDir\Micro
...
+PublishFile $FullBuildOutput\KozaniPackage\KozaniPackage.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniPackage\KozaniPackage.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\KozaniSendToLocal.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToLocal\KozaniSendToLocal.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\KozaniSendToRemote.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSendToRemote\KozaniSendToRemote.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\KozaniSettings.dll $FullPublishDir\Microsoft.WindowsAppRuntime\
+PublishFile $FullBuildOutput\KozaniSettings\KozaniSettings.pdb $FullPublishDir\Microsoft.WindowsAppRuntime\
```
Comment (DrusTheAxe) [inline build/CopyFilesToStagingDir.ps1:78]: Non -winmd artifacts are experimental but I don't see how they're handled differently from non-experimental e.g. KozaniAppGraph.dll is `100% experimental and will be for some time. It should be omitted from preview+stable. How's that indicated in the TransportPackage?

File: build/CopyFilesToStagingDir.ps1 [hunk 3]
```diff
@@ -172,6 +234,8 @@ PublishFile $FullBuildOutput\WindowsAppRuntime_BootstrapDLL\Microsoft.WindowsApp
 # Tools
 PublishFile $FullBuildOutput\WindowsAppRuntime_MSIXInstallFromPath\WindowsAppRuntime_MSIXInstallFromPath.exe $NugetDir\tools\$Platform
 PublishFile $FullBuildOutput\WindowsAppRuntime_MSIXInstallFromPath\WindowsAppRuntime_MSIXInstallFromPath.pdb $NugetDir\tools\$Platform
+PublishFile $FullBuildOutput\MakeMSIX\MakeMSIX.exe $NugetDir\tools\$Platform
+PublishFile $FullBuildOutput\MakeMSIX\MakeMSIX.pdb $NugetDir\tools\$Platform
```
Comment (DrusTheAxe) [inline build/CopyFilesToStagingDir.ps1:238]: This should make its way into the Main or (more likely) Singleton package and have an AppExecutionAlias. How do you define one to make its way into the TransportPackage and be picked up appropriately by the MSIX creation logic in the Aggregator?

**Pull request closed without merging**